src/pages/become-a-sponsor.js

```diff
@@ -248,6 +271,150 @@ export default () => {
           </tbody>
         </table>
 
+        <Styled.h3>A La Carte Sponsorship Options</Styled.h3>
```

Comment:
This section replaces the `Exclusive Sponsorship options`, so we should probably remove it: https://github.com/undergroundjs/website/pull/38/files#diff-c0a411ccae889ddca60b183f7439007fR90-R100